src/store/reducer/view.ts

```diff
@@ -8,6 +8,7 @@ const INITIAL_VIEW_STATE: ViewState = {
   feedbackEnabled: false,
   hotkeysAreActive: true,
   noteFocused: false,
+  hotkeyNotificationsEnabled: typeof window !== "undefined" && window.localStorage.getItem("scrumlr/hotkeyNotificationsEnabled") !== "false",
```

Comment:
Also, here we should also use the `getFromStorage` function provided by `utils/storage.ts`